superset/cli.py

```diff
@@ -628,3 +628,18 @@ def alert() -> None:
             resolution=6000,
             session=session,
         )
+
+
+@superset.command()
+@with_appcontext
+def create_db() -> None:
+    """Resets database"""
+    db.create_all()
+
+
+@superset.command()
+@with_appcontext
+@click.confirmation_option(prompt="Are you sure you want to drop the db?")
+def drop_db() -> None:
```

Comment:
I remember the day where someone ran `airflow reset_db` at a previous job thinking they were connected to their sandbox and they were actually messing with connection strings and were connected to production. That was not a good day, I'm not sure how I feel about this cli subcommand.

Reply:
In addition to adding the db name in the prompt/confirmation, perhaps we can also wrap this in `if app.config["TESTING"]:` to ensure that this doesn't run in a production or dev environment.